CHANGELOG.md

```diff
@@ -12,6 +18,10 @@
 #### OTA for NXP LPC54018 IoT Module
 - Over-the-air (OTA) updates are now available for LPC54018.
 
+#### Ethernet Support on Nuvoton NuMaker-IoT-M487
+- Ethernet porting code has been added for Nuvoton NuMaker-IoT-M487.
```

Comment:
"Ethernet is now available for Nuvoton NuMaker-IoT-M487."

Reply:
Sounds better.